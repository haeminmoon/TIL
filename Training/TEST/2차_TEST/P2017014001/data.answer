Case #1
114
Case #2
430
Case #3
599
Case #4
663
Case #5
778
Case #6
778
Case #7
535
Case #8
511
Case #9
5
Case #10
504
Case #11
970
Case #12
505
Case #13
173
Case #14
519
Case #15
611
Case #16
583
Case #17
833
Case #18
900
Case #19
195
Case #20
415
Case #21
777
Case #22
11
Case #23
489
Case #24
395
Case #25
67
Case #26
476
Case #27
267
Case #28
52
Case #29
317
Case #30
647
Case #31
160
Case #32
960
Case #33
922
Case #34
838
Case #35
625
Case #36
312
Case #37
536
Case #38
546
Case #39
160
Case #40
296
Case #41
160
Case #42
147
Case #43
53
Case #44
227
Case #45
618
Case #46
648
Case #47
968
Case #48
380
Case #49
300
Case #50
687
Case #51
835
Case #52
3
Case #53
239
Case #54
630
Case #55
440
Case #56
542
Case #57
15
Case #58
920
Case #59
703
Case #60
582
Case #61
141
Case #62
394
Case #63
186
Case #64
377
Case #65
620
Case #66
769
Case #67
120
Case #68
545
Case #69
53
Case #70
902
Case #71
384
Case #72
771
Case #73
419
Case #74
677
Case #75
143
Case #76
474
Case #77
104
Case #78
937
Case #79
540
Case #80
819
Case #81
282
Case #82
785
Case #83
286
Case #84
492
Case #85
749
Case #86
388
Case #87
630
Case #88
985
Case #89
458
Case #90
432
Case #91
264
Case #92
458
Case #93
80
Case #94
11
Case #95
6
Case #96
41
Case #97
264
Case #98
565
Case #99
913
Case #100
945
