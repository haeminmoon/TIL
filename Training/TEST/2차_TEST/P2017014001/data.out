Case #1
980
Case #2
792
Case #3
167
Case #4
263
Case #5
11
Case #6
560
Case #7
322
Case #8
265
Case #9
266
Case #10
537
Case #11
133
Case #12
632
Case #13
161
Case #14
912
Case #15
661
Case #16
739
Case #17
128
Case #18
454
Case #19
893
Case #20
247
Case #21
817
Case #22
751
Case #23
224
Case #24
869
Case #25
668
Case #26
125
Case #27
235
Case #28
292
Case #29
79
Case #30
898
Case #31
978
Case #32
274
Case #33
244
Case #34
271
Case #35
463
Case #36
811
Case #37
96
Case #38
510
Case #39
785
Case #40
368
Case #41
355
Case #42
294
Case #43
792
Case #44
122
Case #45
511
Case #46
732
Case #47
976
Case #48
597
Case #49
816
Case #50
82
Case #51
107
Case #52
44
Case #53
509
Case #54
373
Case #55
123
Case #56
616
Case #57
563
Case #58
654
Case #59
660
Case #60
699
Case #61
265
Case #62
783
Case #63
968
Case #64
635
Case #65
545
Case #66
371
Case #67
185
Case #68
304
Case #69
229
Case #70
760
Case #71
227
Case #72
510
Case #73
655
Case #74
491
Case #75
189
Case #76
109
Case #77
420
Case #78
990
Case #79
336
Case #80
601
Case #81
222
Case #82
272
Case #83
243
Case #84
925
Case #85
181
Case #86
64
Case #87
193
Case #88
280
Case #89
833
Case #90
205
Case #91
119
Case #92
168
Case #93
406
Case #94
29
Case #95
7
Case #96
617
Case #97
359
Case #98
990
Case #99
347
Case #100
292
